docs/catalog.md

```diff
@@ -485,13 +485,13 @@ The following is an example of configuring Slack, creating a package binding, an
 2. Create a package binding with your Slack credentials, the channel that you want to post to, and the user name to post as.
 
   ```
-  $ wsk package bind /whisk.system/slack mySlack --param url 'https://hooks.slack.com/services/...' --param username 'Bob' --param channel '#MySlackChannel'
+  $ wsk package bind /whisk.system/slack mySlack --param url "https://hooks.slack.com/services/..." --param username Bob --param channel #MySlackChannel
```

Comment:
really? I didn't see it being dropped since it's already inside a code block with triple back ticks 
No problem I can add the quotes